services/image-registry/README.md

```diff
@@ -0,0 +1,104 @@
+ 
+# Using Keel and Harbor for automated image deployment
+Keel is an operator for tracking image updates and updating Kubernetes resources accordingly. Apart from polling and webhooks notifications for deployment updates (Deployments, StatefulSet, DaemonSet, Helm), it features an approval system through a Slack bot and update notifications integrated with some chat platforms.
+
+It is a much simpler approach than using a full-fledged GitOps solution like [ArgoCD](https://argoproj.github.io/cd/) or [FluxCD](https://fluxcd.io/). We think the a full blow GitOps approach is overkill to our use case and might contribute negatively to the already accumulated technical debt.
+
+> [!WARNING]  
+> We assume that Harbor will take the first IP in the range `172.xxx.255.200` and that Keel will take the second `172.xxx.255.201`. Make sure to pay attention to places where this might be problematic (cluster config, mainly).
+
+## Create the cluster using the config file provided
+This config file allows the kubelet (running containerd) inside the containerised kind nodes to pull from registries with unverified TLS certificates. Check out more in this [StackOverflow post](https://stackoverflow.com/a/65116333).
+
+```bash
+kind create cluster --config image-registry-cd-test-cluster.kind.yaml
+```
+
+## Instal MetalLB
+> [!IMPORTANT]
+> Be sure to replace the xxx's in the template, chech [this page](https://kind.sigs.k8s.io/docs/user/loadbalancer/#setup-address-pool-used-by-loadbalancers) to get the correct subnet addresses.
+
+```bash
+kubectl apply -f https://github.com/metallb/metallb/raw/main/config/manifests/metallb-native.yaml
+
+kubectl wait --namespace metallb-system \
+                --for=condition=ready pod \
+                --selector=app=metallb \
+                --timeout=90s
+
+echo "\
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: example
+  namespace: metallb-system
+spec:
+  addresses:
+  - 172.18.255.200-172.18.255.250
+---
+apiVersion: metallb.io/v1beta1
+kind: L2Advertisement
+metadata:
+  name: empty
+  namespace: metallb-system" | \
+kubectl apply -f -
+```
```

Comment:
The cluster will probably use `Cilium` for networking purposes and load balancing purposes so `MetalLB` will not be needed in the full cluster environment.

However, in a dev environment using kind, do you need to install `MetalLB`?  If so, please specify that in the documentation that `MetalLB` needs to be installed and configured if using `kind` locally.

Reply:
We added it to the documentation and moved the documentation to the Wiki as requested.